### ISSUE TITLE
tanh in the MLP?

### ISSUE THREAD

**Issue by maximecb** [ikostrikov/pytorch-a2c-ppo-acktr#29]
Just a question: you're using tanh in the MLPPolicy. Any reason for this? I thought ReLUs were overall better for backpropagating the gradient, so maybe they should be used in the MLPPolicy as well?

**Comment by ikostrikov** [comment]
ReLUs can be better for very deep networks. For shallow ones (and specifically for continuous control), tanh tend to work better.

**Comment by maximecb** [comment]
Thanks for answering my question
